models/game-summary/buildEnhancedGameSummary.js

```diff
@@ -80,7 +80,7 @@ function buildEnhancedGameSummary(_summary) {
 
 	const casualGame = summary.gameSetting.casualGame;
 	const practiceGame = summary.gameSetting.practiceGame;
-	const unlisted = summary.gameSetting.practiceGame;
+	const unlisted = summary.gameSetting.unlisted;
```

Comment:
I mean ..  you could've made the name of the var consistent with the rest.

Reply:
but that would imply changing it in the other locations `unlisted` is used for consistency and to do to that id need to not use the github editor